quickstart-sda.md

```diff
@@ -0,0 +1,240 @@
+# Quickstart guide to the SDA
+
+This document contains information on how to work with the Sensitive Data Archive (SDA) from the GDI user perspective.
+
+## Brief description of the storage and interfaces stack
+
+The storage and interfaces software stack for the GDI-starter-kit consists of the following services:
+
+| Component     | Description |
+|---------------|------|
+| broker        | RabbitMQ based message broker, [SDA-MQ](https://github.com/neicnordic/sda-mq). |
+| database      | PostgreSQL database, [SDA-DB](https://github.com/neicnordic/sda-db). |
+| storage       | S3 object store, demo uses Minio S3. |
+| mock-oidc     | A python implementation of a mock-oidc in place of LS-AAI. |
+| s3inbox       | Proxy inbox to the S3 backend store, [SDA-S3Proxy](https://github.com/neicnordic/sda-s3proxy). |
+| download      | Data out solution for downloading files from the SDA, [SDA-download](https://github.com/neicnordic/sda-download). |
+| SDA-pipeline     | The ingestion pipeline of the SDA, [SDA-pipeline](https://github.com/neicnordic/sda-pipeline). This comprises of the following core components: `ingest`, `verify`, `finalize` and `mapper`.|
+
+Detailed documentation on the `sda-pipeline` can be found at: https://neicnordic.github.io/sda-pipeline/pkg/sda-pipeline/.
+
+NeIC Sensitive Data Archive documentation can be found at: https://neic-sda.readthedocs.io/en/latest/ .
+
+## Deployment
+
+The storage and interfaces stack can be deployed with the use of the provided `docker-compose.yml` file by running
+
+```shell
+docker compose up -d
+```
+
+from the root of this repo. Configuration can be customized by changing the [`config/config.yml`](./config/config.yml) file.
+
+### Adding TLS to internet facing services
+
+Internet facing services such as `s3inbox`, `download` and `mock-oidc`, need to be secured via TLS certification. This can be most conveniently achieved by using [Let's Encrypt](https://letsencrypt.org/getting-started/) as Certificate Authority. Assuming shell access to your web host, a convenient way to set this up is through installing Certbot (or any other ACME client supported by Let's Encrypt). Detailed instructions on setting up Certbot for different systems can be found [here](https://certbot.eff.org/).
```

Comment:
```suggestion
Internet facing services such as `s3inbox`, `download`, need to be secured via TLS certification. This can be most conveniently achieved by using [Let's Encrypt](https://letsencrypt.org/getting-started/) as Certificate Authority. Assuming shell access to your web host, a convenient way to set this up is through installing Certbot (or any other ACME client supported by Let's Encrypt). Detailed instructions on setting up Certbot for different systems can be found [here](https://certbot.eff.org/).
```

The mock-oidc should't have a letsEncrypt certificate

Reply:
Indeed!